apps/gruenbuch/models.py

```diff
@@ -115,6 +116,15 @@ class GruenbuchOverviewPage(MetadataPageMixin, Page):
     ]
 
 
+class GruenbuchIndexPage(GruenbuchOverviewPage):
+    template = 'gruenbuch_overview_page.html'
```

Comment:
Its the same as parent page, right? So do you need that again?

Reply:
when I didn't have it it automatically looked for a template called index_page which didn't exist